test/spec/vis/vis.spec.js

```diff
@@ -444,43 +446,73 @@ describe("Vis", function() {
 
   describe("dragging option", function() {
 
-    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+    beforeEach(function() {
       var container = $('<div>').css('height', '200px');
-      var vis = new cdb.vis.Vis({el: container});
+      this.vis = new cdb.vis.Vis({el: container});
+    });
 
-      var mapConfig = {
-        updated_at: 'cachebuster',
-        title: "irrelevant",
-        url: "http://cartodb.com",
-        center: [40.044, -101.95],
-        bounding_box_sw: [20, -140],
-        bounding_box_ne: [ 55, -50],
-        zoom: 4,
-        bounds: [[1, 2],[3, 4]],
+    it("should be enabled with zoom overlay and scrollwheel enabled", function() {
+      var mapConfig = createMapConfig({
         scrollwheel: true,
-        overlays: [
-          {
-            type: "zoom",
-            order: 6,
-            options: {
-              x: 20,
-              y: 20,
-              display: true
-            },
-            template: ""
-          }
-        ],
-      };
-
-      vis.load(mapConfig);
-      expect(vis.map.get('drag')).toBeTruthy();
+        zoom: true
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeTruthy();
     });
 
     it("should be enabled with zoom overlay and scrollwheel disabled", function() {
-      var container = $('<div>').css('height', '200px');
-      var vis = new cdb.vis.Vis({el: container});
+      var mapConfig = createMapConfig({
+        scrollwheel: false,
+        zoom: true
+      });
+      this.vis.load(mapConfig);
+      expect(this.vis.map.get('drag')).toBeTruthy();
```

Comment:
The fact that we have to use `this.vis.map` in this assertion might be telling us that it's probably the map the one who has to decide if dragging is enabled based on some of it's attributes (we would need to pass more stuff to the map). Just an observation...

cc// @acanimal